src/Module.zig

```diff
@@ -3514,6 +3528,560 @@ fn semaFile(mod: *Module, file: *File) SemaError!void {
     }
 }
 
+const LowerZon = struct {
+    mod: *Module,
```

Comment:
We're migrating towards the name `zcu: *Zcu` rather than `mod: *Module` -- try to exclusively use this in new code. This file will eventually be renamed to `Zcu.zig`.